bmt/toolkit.py

```diff
@@ -724,6 +724,52 @@ def is_predicate(self, name: str) -> bool:
         """
         return 'related to' in self.get_ancestors(name)
 
+    @lru_cache(CACHE_SIZE)
+    def is_mixin(self, name: str) -> bool:
+        """
+        Determines whether the given name is the name of a mixin
+        in the Biolink Model. An element is a mixin if one of its properties is "is_mixin:true"
+
+        Parameters
+        ----------
+        name: str
+            The name or alias of an element in the Biolink Model
+
+        Returns
+        -------
+        bool
+            That the named element is a valid mixin in Biolink Model
+        """
+        is_mixin = "false"
```

Comment:
Would be good to return actual boolean `False` Instead of just a string “false”.

Because I believe that `element.mixin` actually has a value of `True` If that element is a mixin.

Also, the return value for this method should be either `True` Or `False`. If it is `None`, that might be confusing. Instead, it would be better to just raise a warning that says “Element not found” for scenarios where a given element does not exist.